.github/workflows/benchmark.yml

```diff
@@ -105,6 +106,13 @@ jobs:
           path: ".ccache"
           key: ccache-benchmark-${{ github.sha }}
 
+      - name: "Checkout Contender again"
```

Comment:
nit: Could you just switch the baseline and contender up above?

Reply:
Yeah, not sure why I didn't do that from the get go :D